.ci/make.sh

```diff
@@ -154,12 +154,16 @@ fi
 
 if [[ "$CMD" == "bump" ]]; then
     # Change version to src/Client.php
-    # sed -i "s/const VERSION = '[0-9]\+.[0-9]\+.[0-9]\+\(-dev\)\?'/const VERSION = '$VERSION'/g" $repo/src/Client.php 
+    sed -i "s/const VERSION = '[0-9]\+\.[0-9]\+\.[0-9]\+'/const VERSION = '$VERSION.0'/" $repo/src/Client.php
```

Comment:
I believe `$VERSION` is the complete three-number version (ie `8.3.4`). Might have to update below to match the expected input format.

Reply:
I fixed this extracting a `$MINOR_VERSION` from `$VERSION`. Thanks!